### ISSUE TITLE
Accessibility Support

### ISSUE THREAD

**Issue by DarkPurple141** [SmartCourse/app#45]
A user should be able to rely on w3c standards for accessibility; things like ARIA roles, sensible colour contrasts, alt image tags etc.
